aws/cloudformation/components/ami.yml.erb

```diff
@@ -51,16 +51,22 @@ frontend_device_name ||= '/dev/sda1'
   FrontendInstanceProfile:
     Type: AWS::IAM::InstanceProfile
     Properties: {Roles: [!Ref FrontendRole]}
-  # Signal when the instance is fully provisioned and ready for AMI creation.
+  # The WebServerAMI UserData shell script sends a signal when it completes successfully to this Resource.
   AMICreate<%=commit_hash%>:
+    DependsOn: [WebServerAMI]
```

Comment:
`DependsOn` blocks the creation of this resource, which could potentially lead to a race condition where this WaitCondition is not yet created when the waited-upon resource sends it's signal - makes me nervous to list it here. Would it make sense to instead have this WaitCondition mirror the `DependsOn` of the WebServerAmi resource, so they start creation at about the same time (keeping your timeout relevant)?